content/developer/reference/backend/orm.rst

```diff
@@ -407,6 +407,13 @@ Automatic fields
 
     Raise an Error otherwise.
 
+.. attribute:: Model.display_name
+
+    Name :class:`field <odoo.fields.String>` displayed by default in the web client
```

Comment:
Isn't the display_name a `Char` ?
```python
            add_default('display_name', fields.Char(
                string='Display Name', automatic=True, compute='_compute_display_name'))
```

(thought i added that comment to my review but it seems i didn't :man_facepalming:)